<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVeiw">
    <property name="title">
     <string>Appearance</string>
    </property>
    <widget class="QMenu" name="menuFont_size">
     <property name="title">
      <string>Font size</string>
     </property>
     <addaction name="action8"/>
     <addaction name="action10"/>
     <addaction name="action12"/>
     <addaction name="action14"/>
     <addaction name="action24"/>
     <addaction name="action26"/>
     <addaction name="action32"/>
    </widget>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionDark_2"/>
    </widget>
    <addaction name="menuFont_size"/>
    <addaction name="menuMode"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedu"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_2"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVeiw"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8pt</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>12pt</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>16pt</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>20pt</string>
   </property>
  </action>
  <action name="action24">
   <property name="text">
    <string>24pt</string>
   </property>
  </action>
  <action name="action26">
   <property name="text">
    <string>28pt</string>
   </property>
  </action>
  <action name="action32">
   <property name="text">
    <string>32pt</string>
   </property>
  </action>
  <action name="actionDark_mode">
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
  <action name="actionSet_light_mode">
   <property name="text">
    <string>Set light mode</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo         </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark_2">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRedu">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy_2">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>

